ec2net-functions

```diff
@@ -190,12 +197,18 @@ rewrite_primary() {
 	HWADDR=${HWADDR}
 	DEFROUTE=no
 	EC2SYNC=yes
+  MAINROUTETABLE=${MAINROUTETABLE}
 EOF
   cat <<- EOF > ${route_file}
 	default via ${gateway} dev ${INTERFACE} table ${RTABLE}
-	default via ${gateway} dev ${INTERFACE} metric ${RTABLE}
 	${cidr} dev ${INTERFACE} proto kernel scope link src ${primary_ipv4} table ${RTABLE}
 EOF
+  if [ "${MAINROUTETABLE}" == "yes" ]; then
+    logger --tag ec2net "[rewrite_primary] adding default route to main table for ${INTERFACE}"
+    cat <<- EOF >> ${route_file}
+    default via ${gateway} dev ${INTERFACE} metric ${RTABLE}
+EOF
+  fi
```

Comment:
indentation is off here as well.  You seem to be using space, while the rest of the file is using tabs

Reply:
that's exactly what the issue was. Thanks for pointing it out - fixed